.github/CONTRIBUTING.md

```diff
@@ -302,7 +302,7 @@ the code should be flagged or not. Additionally, making auto-fixes to code in a
 
 This defensive coding in a sniff should, of course, also be safeguarded via tests.
 
-Parse/compile errors can take various forms. Some can be handled without much problems by PHPCS, some can not.
+Parse/compile errors can take various forms. Some can be handled without many problems by PHPCS, some can not.
```

Comment:
"without much problems" is an expression and is correct. "without many problems" changes the meaning of the sentence.